tests/CDN/PantherPortalTest.php

```diff
@@ -20,6 +20,10 @@ class PantherPortalTest extends TestCase
 {
     public function testPortal(): void
```

Comment:
Symfony uses @require annotation can we use this too? 

Also this should be fixed in a separate PR

Reply:
Never heard about it, can you point me to an example?

Reply:
Sure, I am currently on a phone 📱, will have a look later

Reply:
never mind, found it (and added it)

Reply:
Does it work?